benchmarks/perf-hash.js

```diff
@@ -1,31 +1,34 @@
 const Benchmark = require('benchmark');
```

Comment:
This file had a conflict, so when the pre-commit formatting run happened, it was included. No changes to this file that are important.